wolox-books/src/app/screens/auth/auth.component.scss

```diff
@@ -0,0 +1,55 @@
+@import '../../scss/application.scss';

```

Comment:
You should only import the partials you need, not the entire application

Reply:
Perfect 👌